drupal/code/themes/custom/kada/preprocess/taxonomy_term.preprocess.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function kada_preprocess_taxonomy_term(&$variables) {
+  if (
```

Comment:
https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct coding standards for control structure